mobile/native/src/channel_fee.rs

```diff
@@ -51,22 +55,31 @@ impl Subscriber for ChannelFeePaymentSubscriber {
 }
 
 impl ChannelFeePaymentSubscriber {
-    pub fn new() -> Self {
+    pub fn new(channel_manager: Arc<ChannelManager>) -> Self {
         Self {
             open_channel_tx: Arc::new(Mutex::new(None)),
+            channel_manager,
         }
     }
 
     /// Attempts to pay the transaction fees for opening an inbound channel.
     fn pay_funding_transaction_fees(&self, amount_msats: u64) -> Result<()> {
-        let transaction = match self.get_funding_transaction() {
-            Some(transaction) => transaction,
+        let (channel_id, transaction) = match self.get_funding_transaction() {
+            Some((channel_id, transaction)) => (channel_id, transaction),
             None => {
                 tracing::debug!("No pending funding transaction found!");
                 return Ok(());
             }
         };
 
+        tokio::task::block_in_place(|| {
+            tracing::debug!(
+                "Waiting for outbound capacity on channel {}.",
+                hex::encode(channel_id)
```

Comment:
🙃 Slightly prefer using `tracing` fields.
```suggestion
                channel_id = %hex::encode(channel_id),
                "Waiting for outbound capacity on channel",
```